tests/Serializer/JsonSerializationTest.php

```diff
@@ -132,7 +131,6 @@ public function getFirstClassListCollectionsValues()
         return [
             [[1, 2, 3], '[1,2,3]'],
             [[], '[]'],
-            [[1, 'a' => 2], '[1,2]'],
```

Comment:
why this have been removed?

Reply:
if you need some data that can be deserialized, create a new data set please

Reply:
I've splited these tests into separate methods and added support for xml.